lib/loader.js

```diff
@@ -22,11 +22,17 @@ module.exports = function (content, sourceMap) {
     const tags = new Set()
     const file = this.fs.readFileSync(this.resourcePath).toString('utf8')
     const component = compiler.parseComponent(file)
-    component.template && compiler.compile(component.template.content, {
-      modules: [{
-        postTransformNode: node => { tags.add(node.tag) }
-      }]
-    })
+    if (component.template) {
+      const pug = require('pug')
```

Comment:
In both ifs, it's only needed if pug is being used.

Reply:
whoops

Reply:
Done